app/controllers/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ApplicationController < ActionController::API
   # https://github.com/gonzalo-bulnes/simple_token_authentication#make-models-token-authenticatable
-  acts_as_token_authentication_handler_for User, except: [:search, :show, :linkedin, :random], fallback: :exception
+  acts_as_token_authentication_handler_for User, except: [:search, :index, :show, :linkedin], fallback: :exception
```

Comment:
I don't agree with this change. The whitelist of publicly available endpoints (actions) was cemented in a previous branch and should not be altered.

Reply:
I didn't intend to make that change. I think this is a relic of my code being out of sync with the main branch.